### PR TITLE
UDP logger for SBP

### PR DESCRIPTION
**Pull request by mfine**
A simple UDP logger. Sends messages over a UDP address:port pair. There should be some library help around framing out the message instead of doing it explicitly here. Haven't tried this out yet.

/cc @fnoble @denniszollo

### CODE REVIEW

**Comment by fnoble** [comment]
lgtm, @mookerji suggested adding a unit test before like this: https://github.com/swift-nav/pySTR4500/blob/master/test/test_client.py.

File: python/sbp/client/examples/udp.py [hunk 1]
```diff
@@ -15,9 +15,8 @@
 
 from sbp.client.drivers.pyserial_driver import PySerialDriver
 from sbp.client.handler import Handler
+from sbp.client.loggers import UdpLogger
```
Comment (denniszollo) [inline python/sbp/client/examples/udp.py]: Is this going to work?  The file is named udp_logger...

Comment (mfine) [inline python/sbp/client/examples/udp.py]: Oh yea, should be

```python
from sbp.client.loggers.udp_logger import UdpLogger
```

Comment (denniszollo) [inline python/sbp/client/examples/udp.py]: Also, we should make the file executable.

**Comment by mfine** [comment]
Going to bring this in for integration. Will finish the test in a later PR.

**Pull request merged**